cloc-1.64.pl

```diff
@@ -69,16 +69,16 @@ BEGIN
     }
 }
 
-my $HAVE_Algorith_Diff = 0;
+my $HAVE_Algorithm_Diff = 0;
```

Comment:
this might be a third party file that shouldn't be touched. let me know and i'll drop it.